Makefile

```diff
@@ -18,9 +18,14 @@ lab:
 	$(CONDA) .conda/docs; jupyter lab --ip=0.0.0.0 --no-browser --ServerApp.token=deadbeef
 
 # Serve the documentation website.
```

Comment:
how is the make file used? do we have docs for this?

Reply:
No but I added some text to each Makefile in this PR just now.